src/components/checkboxes/_checkbox.scss

```diff
@@ -1,6 +1,6 @@
-$checkbox-input-width: 1.25rem;
-$checkbox-padding: 0.6rem;
-$checkbox-label-vertical-padding: 0.75rem;
+$checkbox-input-width: 22.5px;
+$checkbox-padding: 10.8px;
```

Comment:
I think this could be rounded up to 11px.  The left padding on the label is 44px. Taking away the 22px width of checkbox, leaves 11px padding either side.